v1.1.4 php 7 compatible

celtic34fr opened the pull request:
just some correction about plugins declaration, but the same as my last version

celtic34fr committed:
ajout filtrage source pour suppression des balises style + contenu et title

celtic34fr committed:
ajout traitement des mail html avec image(s) embarquée(s)

celtic34fr committed:
ajout plugin traitement mail avec image(s) embarquée(s)

celtic34fr committed:
multi sender, reciever, BCC and CC

add the possibility to have multiple sender (from), reciever (to), BCC or/and CC

celtic34fr committed:
add example for multiple sender with or without name

celtic34fr committed:
add attachments traitement méthod

celtic34fr committed:
add attachments call traitement

celtic34fr committed:
add default type for attachments

celtic34fr committed:
update method getType

celtic34fr committed:
Correction mail generation without image to insert

celtic34fr committed:
correction for qualify and search embedding image with hostname

celtic34fr committed:
add new features to README

celtic34fr committed:
change composer.json

celtic34fr committed:
refer to main repositopry

celtic34fr committed:
update readme

celtic34fr committed:
correction

celtic34fr committed:
correction

celtic34fr committed:
correction

celtic34fr committed:
correction

celtic34fr committed:
correction ..

celtic34fr committed:
correction ...

mtymek commented on README.md:
```diff
@@ -1,11 +1,7 @@
-[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/mtymek/MtMail/badges/quality-score.png?s=f03d22161755c845d0ce06ab90a67cf4e3e340e0)](https://scrutinizer-ci.com/g/mtymek/MtMail/)
-[![Code Coverage](https://scrutinizer-ci.com/g/mtymek/MtMail/badges/coverage.png?s=d4b2ac636d43b3dd8042675dd32ac8fe2cf4e390)](https://scrutinizer-ci.com/g/mtymek/MtMail/)
-[![Latest Stable Version](https://poser.pugx.org/mtymek/mt-mail/v/stable.png)](https://packagist.org/packages/mtymek/mt-mail)
-[![Total Downloads](https://poser.pugx.org/mtymek/mt-mail/downloads.png)](https://packagist.org/packages/mtymek/mt-mail)
```
Please revert this changes for the PR.

mtymek commented on README.md:
```diff
@@ -44,7 +42,7 @@ Creating message from controller:
 
 $headers = array(
-    'to' => 'johndoe@domain.com',
+    'to' => 'johndoe@domain.com, Jade DOE<jade.doe@domain.net>',
```
Please revert to previous version.

mtymek commented on composer.json:
```diff
@@ -1,7 +1,13 @@
 {
-    "name": "mtymek/mt-mail",
+    "name": "celtic34fr/mt-mail",
```
Please revert to previous version

mtymek commented on src/ComposerPlugin/EmbeddingImages.php:
```diff
@@ -0,0 +1,134 @@
+ * User: gilbert
+ * Date: 21/10/16
+ * Time: 14:05
+ */
```
Remove auto-generated doc-block - it does not bring any useful information.

ojhaujjwal commented on README.md:
```diff
@@ -1,11 +1,7 @@
-[![Code Coverage](https://scrutinizer-ci.com/g/mtymek/MtMail/badges/coverage.png?s=d4b2ac636d43b3dd8042675dd32ac8fe2cf4e390)](https://scrutinizer-ci.com/g/mtymek/MtMail/)
-[![Latest Stable Version](https://poser.pugx.org/mtymek/mt-mail/v/stable.png)](https://packagist.org/packages/mtymek/mt-mail)
-[![Total Downloads](https://poser.pugx.org/mtymek/mt-mail/downloads.png)](https://packagist.org/packages/mtymek/mt-mail)
+It's a fork of mtymek/mt-mail package
```
seriously?

mtymek reviewed (changes requested):
I added a few comments here and there regarding some details. However, what I'm missing is:
- some explanation in the PR: what are the new features and why you introduced them
- documentation in `doc` directory
- unit tests for new features

ojhaujjwal commented on src/ComposerPlugin/PlaintextMessage.php:
```diff
@@ -25,7 +25,22 @@ class PlaintextMessage extends AbstractListenerAggregate implements PluginInterf
     private function br2nl($string)
     {
-        return preg_replace('/\<br(\s*)?\/?\>/i', "\n", $string);
+        /* suppression de la balise title */
```
only english please

celtic34fr committed:
remove french comments
add some documentations informations about EmbeddingImage plugin and new way to parameter the recipients of an email

celtic34fr committed:
add reference inside document

celtic34fr committed:
update inside reference (fault in the name)

celtic34fr committed:
update

ojhaujjwal commented:
Please add proper information in the PR description and the tests(mandatory). And some docs would be great as well.

celtic34fr committed:
update

celtic34fr commented:
my latest changes : 
remove the french comments
add some information about new plugin EmbeddingImages and how to declare multiple recipients for an email

for UnitTest i don't see how to set it to be efficient. For PlaintextMessage plugin, my changes affects the message rendered before creating the mail in text format. For EmbeddingImages plugin, the UnitTest is difficult to set up as i know. Could help me in that case ?

celtic34fr commented:
as I read, now MtMail is Php 7.2 compatible, this pulls request is full useless. I close it
